### DISCUSSION TITLE
Should www be added via --add-alias-domains?

### DISCUSSION THREAD

**Discussion by ssuess** [EasyEngine/easyengine#1626] [Support Requests]
Just looking for confirmation on the appropriate way to add www for any domain. If I have a domain `example.com` setup, the best way to make `www.example.com` work is to add it as an alias? What if I just want `www.example.com` to be redirected to `example.com` but don't care about actually serving up the `www.example.com` url itself?

**Comment by tmsbrasil** [comment]
I used this command to solve it ee site update example.com --add-alias-domains='www.example.com'
